### ISSUE TITLE
Alpine "/src" write permission error

### ISSUE THREAD

**Issue by DeviaVir** [floryn90/docker-hugo#147]
**Describe the bug**
Attempts to use the `floryn90/hugo:0.134.3-ext-alpine-onbuild` fail with a write error.

**To Reproduce**
1. Create a Dockerfile relying on the alpine onbuild image
2. Build to an nginx or similar
3. See error

Dockerfile:
```
FROM floryn90/hugo:0.134.3-ext-alpine-onbuild AS hugo
FROM nginx
COPY --from=hugo /target /usr/share/nginx/html
```

**Expected behavior**
Build completing without `/src` permission errors

**Screenshots**
![Screenshot 2024-09-24 11 08 43](https://github.com/user-attachments/assets/647cb372-d22e-4e39-816c-e10501a57ebf)

**Additional context**
Replacing "alpine" with "debian" resolves the issue.
![Screenshot 2024-09-24 11 05 23](https://github.com/user-attachments/assets/b7113594-79bd-4d2e-bd3b-0851776fbec8)

**Comment by DeviaVir** [comment]
Guessing related to these commits?

* https://github.com/floryn90/docker-hugo/commit/98f65a09ccb880f017c4b630261c0f0253417ca6
* https://github.com/floryn90/docker-hugo/commit/54d187233d91b4954e95b9eebcb660eb447cc8ac

**Comment by floryn90** [comment]
Should be solved with #172 
Let me know if you’re still facing the permissions issue
